jvcl/run/JvDBGridExport.pas

```diff
@@ -178,6 +179,7 @@   TJvDBGridExcelExport = class(TJvCustomDBGridOleExport)
     property Visible: Boolean read FVisible write FVisible default False;
     property Orientation: TWordOrientation read FOrientation write FOrientation default woPortrait;
     property AutoFit: Boolean read FAutoFit write FAutoFit;
+    property HeaderRow: Boolean read FHeaderRow write FHeaderRow;
```

Comment:
Specify the default here too:

`property HeaderRow: Boolean read FHeaderRow write FHeaderRow default True;`

Reply:
done, thanks